config/install/firebase.settings.yml

```diff
@@ -1,4 +1,4 @@
 # These are the default values of the configuration for Firebase.
 
 server_key: ''
-endpoint: 'https://fcm.googleapis.com/fcm/send'
+sender_id: ''
```

Comment:
this default looks fine

Reply:
as for me, "endpoint" in config is better, than "const"
for example, for have another url for api v1

Reply:
no, we can't move on api v1 after just changing endpoint and in this case const is better than store this endpoint in config, he anyway always will be same